index.html

```diff
@@ -33,5 +42,139 @@ <h3>I'm a software Developer, </h3>
         </ul>
       </div>
     </section>
+    <section class="section-two">
+      <h1>My Recent Works</h1>
```

Comment:
- [ ] Great job on adding the section title! However, there are a few adjustments that need to be made:
  - [ ] It's recommended to use a more appropriate heading tag for the section title. While using `<h1>` is allowed, it's generally recommended to have only one `<h1>` tag per webpage for better accessibility and clarity. Consider using a heading tag that reflects the hierarchy and relevance of the title within your document, such as `<h2>` to `<h6>`. This will ensure a logical heading structure and improve accessibility for users.
  
Please make these adjustments to improve the accessibility and visual consistency of your section title. Keep up the good work! 👏🌟